pyartcd/pyartcd/pipelines/release_from_fbc.py

```
@@ -0,0 +1,729 @@
+            # Use doozer command to read group config product field
+            doozer_cmd = ['doozer', f'--group={self.group}', 'config:read-group', 'product']
+
+            _, product_output, _ = await exectools.cmd_gather_async(doozer_cmd)
+            # Clean up the output - remove all whitespace (including newlines)
+            product = product_output.strip()
+
+            if product and product != 'None' and product != 'null':
+                self.logger.info(f"Loaded product from group config: {product}")
+                return product
+            else:
+                self.logger.debug("No product field found in group config, falling back to group name extraction")
```

Pylint (4.0.2)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)